tf2x.yml

```diff
@@ -0,0 +1,15 @@
+name: tf2.10X
```

Comment:
First of all, this fancy name will not work. It needs to be human readable like `dmri_seg` and this recipe needs to replace `environment_gpu.yml`.